### DISCUSSION TITLE
Frameless window's size is not right

### DISCUSSION THREAD

**Discussion by JesuisTong** [tauri-apps/tauri#11881] [Q&A] [answered]
When `decorations: true`, HTML size is correct, but set to `false`, height always 14px heigher, and width 36px heigher.

----
Edit, seems like Electron has the same problem, maybe it's kind of `feature`?

**Comment by FabianLars** [comment, marked as answer]
this should be fixed on the next release iirc, but interesting that electron has that problem too.

**Reply by JesuisTong** [reply]
Ok, thx a lot.
In Electron set `frame: true`, width and height are both lower than what you have set, but `frame: false` is correct. LOL

**Reply by ShawnRZ** [reply]
Btw, maxSize and minSize are not right too. Width always 16px lower, and height 9px lower.
